ipaserver/plugins/host.py

```diff
@@ -730,6 +730,9 @@ def pre_callback(self, ldap, dn, entry_attrs, attrs_list, *keys, **options):
         update_krbticketflags(ldap, entry_attrs, attrs_list, options, False)
         if 'krbticketflags' in entry_attrs:
             entry_attrs['objectclass'].append('krbticketpolicyaux')
+        if 'krbprincipalauthind' in entry_attrs:
+            validate_auth_indicator(entry_attrs)
```

Comment:
I think you can safely always run this validator as the same check is done at lower levels.